CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ All notable changes to this project are documented in this file.
 ### Widgets
 
  - Checkbox: fix text color in fluent style (#6239)
+ - TextInput: fix selection colors not used from style (#6326)
```

Comment:
Technically, TextInput is not a widget, it's a builtin element, so i'd put that into Slint language (although I agree the difference is thin)

Please put the changelog entry in the commit message when you squash instead of editing the CHANELOG.md file in order to avoid conflicts.